src/componentes/header.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import coffee_logo from '../photo/coffee-logo.png';
+import shopping_cart from '../photo/shopping-cart.png';
+import profile_image from '../photo/profile-image.png';
+import menu_icon from '../photo/menu-icon.svg';
+class Header extends Component {
+    state = {  }
+    onHeaderItemClick (){
+      console.log("clicked");
```

Comment:
Need to complete this method
```suggestion
      console.log("clicked");
```